LightFramework/src/rendering/editorcamera.cpp

```diff
@@ -10,30 +10,30 @@
 namespace Light
 {
 	EditorCamera::EditorCamera(float fovy, float aspectRatio, float near, float far)
-		:	m_aspectRatio(aspectRatio),
-			m_fovy(fovy),
-			m_near(near),
-			m_far(far)
+		: m_aspectRatio(aspectRatio),
+		  m_fovy(fovy),
+		  m_near(near),
+		  m_far(far)
 	{
 		updateView();
 		updateProjection();
 	}
 
 	void EditorCamera::setViewportSize(uint32_t width, uint32_t height)
 	{
-        m_viewportWidth = width;
-        m_viewportHeight = height;
-        m_aspectRatio = float(width) / height;
+		m_viewportWidth = width;
+		m_viewportHeight = height;
+		m_aspectRatio = float(width) / height;
```

Comment:
I know it had a C style cast before also, but maybe it would be better to have a C++ style cast here.